.gitignore

```diff
@@ -1,25 +1,18 @@
 .vscode
 .vagrant
 .DS_Store
-build
```

Comment:
A handful of changes here but ultimately to fix the problem with arbitrary `build` directories under vendor not being inlcuded.